### PR TITLE
Create Carousel Option 2 

### PR DESCRIPTION
**Pull request by johnang-del**
Carousel Option 2 for Thumbnails

### CODE REVIEW

**Comment by johnang-del** [comment]
Screen Shot 2019-10-01 at 9 33 58 AM

File: src/components/Carousel/Carousel.js [hunk 1]
```diff
@@ -35,6 +35,32 @@ const Slide = ({ image }) => {
 	);
 };
 
+const Thumb = ({ image }) => {
+	const styles = {
+		width: '33.33%',
+		height: '100%',
+		display: 'inline-block'
+	};
+	const childStyle = {
```
Comment (pauvalenzuela) [inline src/components/Carousel/Carousel.js]: we can use our styles directory for this one.

File: src/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -62,17 +95,25 @@ class Carousel extends Component {
...
-		}));
+		if (!this.props.thumbnails) {
+			this.setState((prevState) => ({
+				currentIndex: prevState.currentIndex + 1,
+				translateValue: prevState.translateValue + -this.slideWidth()
+			}));
+		} else {
+			this.setState((prevState) => ({
+				currentIndex: prevState.currentIndex === 0 ? 3 : prevState.currentIndex + 1,
+				translateValue: prevState.translateValue + -(this.slideWidth() / 3)
```
[the remark this answers is not in the document]
Comment (johnang-del) [reply src/components/Carousel/Carousel.js]: i spoke to Randell about the design and adjustment on the number of images and aesthetically it works for 3, we can maybe revisit later if the need arises on image number adjustments

[the remark this answers is not in the document]
Comment (pauvalenzuela) [reply src/components/Carousel/Carousel.js]: Noted on this one.

**Pull request merged**